README.md

```diff
@@ -1,29 +1,92 @@
-# Tuples 
+Learn Tuples with Swift
+======================
 
+We've come a long way to learn the fundamentals of Swift. With our knowledge of variables, `Strings`, `Booleans` and collections, we can now create *functions*.  Functions are the building blocks of any application - large or small. To introduce the new (and funny-sounding) idea of `tuples`, let's review how functions can be used to send and receive data.
 
```

Comment:
This is a great introduction. I'm always a fan of reintroducing what it is they currently know as well as introduce the new concept they are about to take on.